attributes/default.rb

```diff
@@ -52,3 +54,10 @@
 default['onlinefs']['kafka_consumer']['topic_pattern']    = ".*_onlinefs"
 default['onlinefs']['kafka_consumer']['topic_list']       = ""
 default['onlinefs']['kafka_consumer']['poll_timeout_ms']  = 1000
+default['onlinefs']['kafka_consumer']['ron_db_group_id']     = "0"
+default['onlinefs']['kafka_consumer']['vector_db_group_id']  = "1"
+
+# Opensearch
+default['onlinefs']['opensearch']['host']       = "https://elastic.service.consul:9200"
```

Comment:
No, you can't do this. You need to use:
```
consul_helper.get_service_fqdn("elastic")
```
as `.service.consul` is something users can customize so we can't hardcode in the configuration.

Same for the port. There is a chef attribute to overwrite the port, since you are including the `elasticsearch` attributes, use it, don't hardcode the port.